### ISSUE TITLE
Refactor, Streamline, Sync, Tidy up, document infrastructure

### ISSUE THREAD

**Issue by motey** [covidgraph/documentation#90]
At the moment we have 3 Host:

* DEV - Was DEV DB but is used now for running the pipeline atm. 
* COMPUTE - Was the pipeline runner but is running the DEV DB now
* PROD - This is used to run the PROD DB

Every instance has different directories, setups and config styles. 

* [ ] Define a global setup/config style
* [ ] Implement this setup/config on all instances
* [ ] Adapt (and improve) the documentation to these changes

**Comment by motey** [comment]
i am in favor of a all dockerized and traefik proxied approach
and put everything in `/opt/covidgraph`

@mpreusse you are cool with this?

**Comment by motey** [comment]
i want to try to use the treafik tls+tcp-endpoint feature to prevent any need for fumbling with certificates

**Comment by motey** [comment]
we could git the whole config in our ~~covidgraph/steering-group~~ https://github.com/covidgraph/infrastructure-config repo

**Comment by motey** [comment]
@Jiros you created the steering group repo. does every team member has access to https://github.com/covidgraph/infrastructure-config ? what do we need to do to restrict access to only some team members?

**Comment by motey** [comment]
think i found it https://github.com/covidgraph/infrastructure-config/settings/access

**Comment by Jiros** [comment]
@motey Yes, that's now restricted by the look of it. The other option is to set up an Infrstructure team in Github and add people and permissions to that but there's not much difference (other than being able to mention @team in comments etc) and makes more sense for larger setups.

Also, be aware that we can't add private repos to the ZenHub board but that doesn't sound like it will be an issue here.

We can discuss custom label types that might make more sense in that repo. (e.g. Type: Server)
